README.md

```diff
@@ -64,7 +64,7 @@ The frontend will be accessible at `http://localhost:3000/` and the swagger conf
 
 ## Development mode
 
-To use the dev mode add the `http://localhost:3000` prefix to the servers url in `tdm-be/src/swagger/swagger.js`.
+To use the dev mode add the `http://localhost:3000` host in `tdm-be/bin/swagger.ts`.
```

Comment:
<!-- Quibbles are very much like nitpick:, except it does not conjure up images of lice and animal hygiene practices.  -->
**quibble (non-blocking):** ‏Say that it's a variable

```suggestion
To use the dev mode, add set `HOST` to `http://localhost:3000` in `tdm-be/bin/swagger.ts`.
```